avocado/core/nrunner.py

```diff
@@ -557,29 +557,32 @@ class PythonUnittestRunner(BaseRunner):
 
      * kwargs: not used
     """
-    @staticmethod
-    def _uri_to_unittest_name(uri):
+
```

Comment:
In the commit message, I think you meant "can resolve either a module, ...".  IIUC, the "to" changes the meaning, referring to what will be the output, and not the input.